Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs

```diff
@@ -0,0 +1,84 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class FirstPersonInteractor : MonoBehaviour
+{
+	public float interactionRange = 3;
+
+	private Camera viewpoint;
+
+	private Interaction[] avaliableInteractions;
+	private bool interactionAvaliable
+	{
+		get { return avaliableInteractions.Length > 0; }
```

Comment:
I guess this is just a C# thing, but can we talk real quick about the `get` syntax tonight?